### PR TITLE
Continuing work on add new review

### PR DESCRIPTION
**Pull request by RobStafford**
This primarily continues to add functionality to add new review. About 70% of the interface is built. The api call and server endpoint should be working (but are untested). Still haven't attempted picture upload.

### CODE REVIEW

File: client/src/components/Ratings/Ratings.jsx [hunk 1]
```diff
@@ -96,13 +105,23 @@ class Ratings extends React.Component {
...
+  }
+
   render() {
 
     return (
       <div data-testid="ratings" className="ratings-widget">
         <a id="reviews-link"></a>
         <h3>RATINGS & REVIEWS</h3>
-        <br></br><RatingsMeta className="ratings_meta" ratings_meta={this.state.ratings_meta}></RatingsMeta><RatingsList className="ratings_list" ratings_meta={this.state.ratings_meta} count={this.state.count} chooseHelpful={this.chooseHelpful} putMarkHelpful={this.putMarkHelpful} getAllReviews={this.getAllReviews.bind(this)}></RatingsList>
+        <br></br><RatingsMeta className="ratings_meta" ratings_meta={this.state.ratings_meta}></RatingsMeta><RatingsList className="ratings_list" ratings_meta={this.state.ratings_meta} count={this.state.count} item_name={this.state.item_name} chooseHelpful={this.chooseHelpful} putMarkHelpful={this.putMarkHelpful} getAllReviews={this.getAllReviews.bind(this)}></RatingsList>
```
Comment (DominicMonares) [inline client/src/components/Ratings/Ratings.jsx]: line should be split, currently has a very high character count

File: client/src/components/Ratings/components/InputProductBreakdowns.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+      return (
+        <ul className="list">
+          {listItems}
+        </ul>
+      );
+  }
+}
+
+export default InputProductBreakdowns;
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/InputProductBreakdowns.jsx]: newline needed

File: client/src/components/Ratings/components/RatingsList.jsx [hunk 3]
```diff
@@ -99,7 +100,7 @@ class RatingsList extends React.Component {
             <ul className="ratings_list">{ratings}</ul>
           </div>
           <div>
-            <span>{(count_to_display < recommend_total) ? <div><button className="btn" onClick={this.moreReviews}>MORE REVIEWS</button></div>: null}<NewReview className="new_review"></NewReview></span>
+            <span>{(count_to_display < recommend_total) ? <div><button className="btn" onClick={this.moreReviews}>MORE REVIEWS</button></div>: null}<NewReview item_name={this.props.item_name} ratings_characteristics={this.props.ratings_meta.characteristics} className="new_review"></NewReview></span>
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/RatingsList.jsx]: line should be split, currently has a very high character count

**Review by DominicMonares — CHANGES_REQUESTED** [review]
This one looks great as well! Noticed a few minor things and made comments. Looks like there is an unresolved error as well. Good to go other than that!

**Comment by RobStafford** [comment]
Changes made. Thanks Dom.

**Comment by DominicMonares** [comment]
Thanks Rob! Approved and merged.

**Pull request merged**